users/captcha.py

```diff
@@ -0,0 +1,43 @@
+import random
+
+def diceCaptcha():
+    dice = [4,6,8,10,12,20]
+    d1, d2 = random.choice(dice), random.choice(dice)
+    question = ["What is ",
+            random.choice(["the maximum total ", "the largest sum "]),
+            "when rolling ",
+            "a d{}".format(d1),
+            random.choice([" and a ", " plus a "]),
+            "d{}?".format(d2)
+            ]
+    chelp = "(dX means an X-sided die, e.g. d4 means a four sided die with numbers 1 to 4 on it.)"
```

Comment:
chelp is an odd name
also "four sided" -> "four-sided" pls

Reply:
Fixed.